### PR TITLE
Fixes #28387 - Update Rubocop to 0.75

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -34,9 +36,6 @@ Style/StringLiterals:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-Style/FormatStringToken:
```
Comment (mmoll) [inline .rubocop.yml]: See line 98/102

File: .rubocop.yml [hunk 2]
```diff
@@ -87,9 +86,6 @@ Style/IfUnlessModifier:
 Style/NumericPredicate:
   Enabled: false
 
-Style/IfUnlessModifier:
```
Comment (mmoll) [inline .rubocop.yml]: See line 83/84

**Comment by mmoll** [comment]
Test failure unrelated, I guess. Rubocop is fixed.

**Comment by adamruzicka** [comment]
Thank you @mmoll !

**Pull request merged**